docs/reference/execute-methods.md

```diff
@@ -702,6 +702,76 @@ Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 style | string | yes | Either `light` or `dark` | dark
 
+### mobile: getIncreaseContrast
```

Comment:
I would rather use more human-readable execute method names, for example `mobile: increaseContrast`